NetUtilities/System/Int24.cs

```diff
@@ -25,7 +25,7 @@ public Int24(uint value) : this((int)value)
 
         public Int24(int value) 
         {
-            if ((uint)value > MaxValue)
+			if ((uint)parsed > MaxValue) or ((uint)parsed < MinValue)
```

Comment:
The parsed value is already being casted to `uint`, so it will always be positive. It makes no sense to compare it with `MinValue` since it can't be negative to start with.